### ISSUE TITLE
Sweeping edge doesn't work.

### ISSUE THREAD

**Issue by xelotex** [CammiePone/Cammies-Combat-Tweaks#13]
Sweeping edge enchantment doesn't work in 1.18.

**Comment by CammiePone** [comment]
:thocc:

lemme give it a go and see. that's really weird tho

**Comment by CammiePone** [comment]
Can't reproduce the bug on its own. Maybe it's a mod conflict?

**Comment by xelotex** [comment]
> Can't reproduce the bug on its own. Maybe it's a mod conflict?

it could be, let me see

**Comment by xelotex** [comment]
> Can't reproduce the bug on its own. Maybe it's a mod conflict?

yes, it was a conflict, but i dont know with what mod its conflicting with.

**Comment by CammiePone** [comment]
well, if you can find it lemme know. odds are it's their fault, but I'm not gonna rule out the possibility I did a fucky wucky somewhere.

**Comment by xelotex** [comment]
I haven't found any incompatibilities yet, but look at this, the sweeping edge will randomly work on some worlds and it wont on some others, really weird, haven't you experienced this bug yet? it seems hard to detect what's causing it, it just randomly happens
file:///C:/Users/Steer%20Brother's/Videos/Captures/Minecraft_%201.18%20-%20Singleplayer%202021-12-09%2019-49-52.mp4

**Comment by CammiePone** [comment]
That, uh, didnt upload the video

**Comment by xelotex** [comment]
its a link, you need to copy and paste it on your browser, you know it right?

**Comment by CammiePone** [comment]
That's not a url, that's a directory on your own local computer.

**Comment by xelotex** [comment]
ohh i see,  i'm dumb.

**Comment by xelotex** [comment]
Now look:
https://www.youtube.com/watch?v=oHPvTggBWEw

**Comment by CammiePone** [comment]
Huh... weird. Not actually sure why that's happening. If I had to guess, maybe it has something to do with the surroundings? I have checks with Sweeping Edge to ensure that it can't hit through blocks, and I thought I had thoroughly tested it, but it seems maybe not?

**Comment by CammiePone** [comment]
Or it's something else entirely, but idk

**Comment by xelotex** [comment]
no, the problem are not the blocks, i tried to hit that zombie multiple times without any block in my path, it should be something else, but no clue what could be, it just randomly happens, maybe one day we will know, but anyways thanks.

**Comment by CammiePone** [comment]
That doesn't mean _my_ checks aren't faulty. The blocks could still be at fault. Can you try in a flat, open area on the worlds where it doesn't work for me?

**Comment by xelotex** [comment]
you were right.
https://www.youtube.com/watch?v=0rU9YAVVFDM

**Comment by CammiePone** [comment]
Alright, cool. So it is something I did.
